qm_hw:
  loop_frequency: 800 
  cycle_time_error_threshold: 0.002 
  thread_priority: 95
  power_limit: 9 
  contact_threshold: 40 
